src/libraries/Microsoft.PowerFx.Json/Functions/JsonFunction.cs

```diff
@@ -0,0 +1,262 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Drawing;
+using System.Globalization;
+using System.IO;
+using System.Text;
+using System.Text.Json;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.PowerFx.Core.Entities;
+using Microsoft.PowerFx.Core.Functions;
+using Microsoft.PowerFx.Core.IR;
+using Microsoft.PowerFx.Core.Types.Enums;
+using Microsoft.PowerFx.Functions;
+using Microsoft.PowerFx.Types;
+
+namespace Microsoft.PowerFx.Core.Texl.Builtins
+{
+    internal class JsonFunctionImpl : JsonFunction, IAsyncTexlFunction4
```

Comment:
>JsonFunction [](http://example.com/codeflow?start=38&length=12)

? Do we need this base class?

Reply:
Yes, we share/use the code and it follow ParseJson pattern